config/ubuntu/waagent.conf

```diff
@@ -82,6 +82,9 @@ OS.SshDir=/etc/ssh
 # Enable RDMA management and set up, should only be used in HPC images
 # OS.EnableRDMA=y
 
+# Enable RDMA kernel update, should only be used with Ubuntu
```

Comment:
This option can be used for other distributions (although they are not checked in the code today). The comment there is for not confusing people running on other distributions.